nvtabular/ops/normalize.py

```diff
@@ -46,24 +47,31 @@ def __init__(self):
         super().__init__()
         self.means = {}
         self.stds = {}
+        self.means_cupy = self.stds_cupy = None
 
     @annotate("Normalize_fit", color="green", domain="nvt_python")
     def fit(self, columns: ColumnNames, ddf: dask_cudf.DataFrame):
         return _custom_moments(ddf[columns])
 
     def fit_finalize(self, dask_stats):
+        self.means_cupy = cupy.array(dask_stats["mean"].values)
+        self.stds_cupy = cupy.array(dask_stats["std"].values)
```

Comment:
Perhaps we could do this lazily (and cache the result) when `transform` is first called?  This would make it easier to dispatch between cpu and gpu-specific code, because we would know the type of the gdf.

Reply:
thats a good idea